abci/types/messages_test.go

```diff
@@ -65,19 +65,95 @@ func TestWriteReadMessage(t *testing.T) {
 			Height:  4,
 			ChainID: "test",
 		},
-		// TODO: add the rest
+		// Expanded test cases covering various ABCI message types
```

Comment:
what motivated this change? Did you see a GH issue that mentioned increasing test coverage here?

AFAIK these methods and types are consistent with upstream CometBFT v0.46.x so this change can instead target upstream and then we can pull it here.

Ref: https://github.com/celestiaorg/celestia-core?tab=readme-ov-file#contributing